docs/UserGuide.md

```diff
@@ -45,49 +53,41 @@ MODPRO is a desktop application which helps NUS students in tracking the progres
 
 ## Tasks-related Features
 
-### Adding task functions [Coming Soon in v1.2]
-Adds task into task list
+### Adding a task
+Adds task into task list.
 
-Format: `add TASK`
-* Adds a task to the end of the task list
-* Task refers to the task description to be shown
-* If the task description is empty, an error message will be displayed to inform the user to enter a description
+Format: `t add m/MODULE d/DESCRIPTION`
+* Adds a task to the task list.
+* MODULE refers to the module which the task belongs to.
+* DESCRIPTION refers to the task description to be shown.
+* MODULE has to exist in the module list and be a valid module with at least 6 characters long with the first two being alphabetic characters. Otherwise, an error message will be displayed.
+* If DESCRIPTION is empty, an error message will be displayed.
 
-Examples:
-
-`add CS2105 Assignment 1` adds CS2105 Assignment 1 task into task list
+Example:
 
-`add CS2103T Quiz` adds CS2103T Quiz task into task list
+`t add m/CS2105 d/Assignment 1` adds a task with the module as 'CS2105' and description as 'Assignment 1' into the task list.
 
-### Tagging priority of task [Coming Soon in v1.2]
-Tags the priority to complete task 
+### Deleting task function [Coming Soon in v1.2]
+Deletes the specified task according to the index given
 
-Format: `tag TASK /p PRIORITY_STATUS`
-* Tags one of three priorities("high", "medium", "low") to the task
-* If the priority status is not "high", "medium" or "low", an error message will appear to inform the user to key in a valid priority status
+Format: `delete INDEX`
+* Deletes the task at the specified index from the storage
```

Comment:
I think you should specify what is specified index so in this case it would be the index in the displayed task list.